parser/tests/Index.test

```diff
@@ -37,9 +37,18 @@ CREATE INDEX test ON test
                          ^
 ==
 
-# Currently supported indexes (bloom_filter)
+# Currently supported indices (csbtree, bloomfilter)
+# Unsupported indices should result in syntax error
 CREATE INDEX test ON test USING arbitrary
 --
 ERROR: syntax error (1 : 33)
 CREATE INDEX test ON test USING arbitrary
                                 ^
+==
+CREATE INDEX bloomIndex ON test USING BLOOMFILTER
+--
+CreateIndexStatement[index_name=bloomIndex,relation_name=test,index_type=2]
```

Comment:
For `index_type`, could we print sth like `bloom-filter` instead of the magic number `2`?